goblint.opam

```diff
@@ -21,6 +21,7 @@ bug-reports: "https://github.com/goblint/analyzer/issues"
 depends: [
   "dune" {>= "3.6"}
   "ocaml" {>= "4.10"}
+  "dune" {>= "3.0"}
```

Comment:
Why do we have 2 `dune` versions?

Reply:
I think this could be happening when running `make setup` or `make setup-flambda` locally sometimes, depending on the `opam` configuration of the given machine